File: tests/fixtures/agents/claude_agent_sdk_app.py
"""Fixture: Claude Agent SDK (Python). Not real app code."""
from claude_agent_sdk import ClaudeAgentOptions, ClaudeSDKClient, tool


@tool("delete_record", "Delete a database record", {"id": str})
async def delete_record(args):
    return {"content": [{"type": "text", "text": "deleted"}]}


options = ClaudeAgentOptions(allowed_tools=["delete_record"])
client = ClaudeSDKClient(options=options)

File: tests/fixtures/agents/claude_agent_sdk_app.ts
// Fixture: Claude Agent SDK (TypeScript). Not real app code.
import { ClaudeSDKClient, tool } from "@anthropic-ai/claude-agent-sdk";

const transferFunds = tool({ name: "transfer_funds", description: "move money" });

const client = new ClaudeSDKClient({ tools: [transferFunds] });

export { client };
